code/controllers/subsystem/ticker.dm

```diff
@@ -191,36 +194,73 @@ SUBSYSTEM_DEF(ticker)
 				send_tip_of_the_round()
 				tipped = TRUE
 
+			if(timeLeft <= 300 && !pre_setup_completed)
```

Comment:
okay so, I don't think this is a good approach. Pre setup should always run at least once to have setup not fail on null mode. There isn't really a need to track if this exact part of setting up failed

Reply:
Pre setup runs only once. pre_setup_completed gets set to true if its successful so it doesnt get run again. There is a check in the normal game start that checks if pre setup was completed so it doesnt run twice

Reply:
Yes, nevermind that, loading rebe takes a while so it's the best solution